docker-compose.yml

```diff
@@ -10,5 +7,34 @@ services:
         - https_proxy
         - no_proxy
     ports:
-      # TODO: change this mapping to map the port your application listens on
       - '3209:3209'
+    depends_on:
+      pcs-frontend-cache:
+        condition: service_healthy
+    environment:
+      - REDIS_CONNECTION_STRING=redis://pcs-frontend-cache:6379
+    networks:
+      - frontend-network
+
+  pcs-frontend-cache:
+    image: bitnami/redis:7.4.1
```

Comment:
We are on Azure Redis version 6, but i don't think it might cause any issues. @MoLala1 ?

Reply:
it technically shouldnt be an issue as in charts.yaml: https://github.com/hmcts/pcs-frontend/blob/1cf213dd7bf0529f7522fd62a606353cf6ea9615/charts/pcs-frontend/Chart.yaml#L11

am using image version 20.3.0 but please test to be sure Jake :-)

Reply:
It's not an issue currently, but we are developing against a latest version in local and someone might make a change and it to realise it won't work in Azure as azure instances are o old version. But happy to try out and change if we see such issues in future.